packages/relay-test-utils/RelayMockPayloadGenerator.js

```diff
@@ -391,33 +462,7 @@ class RelayMockPayloadGenerator {
     ) {
       value = defaultValues[applicationName];
 
-      // Let's validate the correctness of the provided enum value
-      // We will throw if value provided by mock resolvers is invalid
-      if (
-        enumValues !== null &&
-        !enumValues
-          .map(s => s.toUpperCase())
-          .includes(String(value).toUpperCase())
-      ) {
-        invariant(
-          false,
-          'RelayMockPayloadGenerator: Invalid value "%s" provided for enum ' +
-            'field "%s" via MockResolver.' +
-            'Expected one of the following values: %s.',
-          String(value),
-          `${path.join('.')}.${applicationName}`,
-          enumValues.map(v => `"${v}"`).join(', '),
-        );
-      }
-
-      // But missing case should be acceptable, we will just use
-      // a correct spelling from enumValues
-      if (enumValues !== null && value !== undefined) {
-        const correctSpellingEnumIndex = enumValues
-          .map(s => s.toUpperCase())
-          .indexOf(String(value).toUpperCase());
-        value = enumValues[correctSpellingEnumIndex];
-      }
+      value = this._getCorrectDefaultEnum(enumValues, value, path, applicationName);
```

Comment:
and here we would call it `if (enumValues != null) { ...` which will provide more context on why we're calling `_getCorrectDefaultEnum(...)`